src/controller/python/templates/python-CHIPClusters-cpp.zapt

```diff
@@ -37,9 +37,34 @@ void OnDefaultFailureResponse(void * /* context */, uint8_t status)
         gFailureResponseDelegate(status);
 }
 
-template<class AttributeType>
-void OnAttributeResponse(void * /* context */, AttributeType)
+
+template <class AttributeType>
+void OnAttributeResponse(void * /* context */, AttributeType value)
+{
+    std::string strValue = std::to_string(value);
+    ChipLogProgress(Zcl, "  attributeValue: %s", strValue.c_str());
+    if (gSuccessResponseDelegate != nullptr)
+        gSuccessResponseDelegate();
+}
+
+template <>
+void OnAttributeResponse<chip::ByteSpan>(void * /* context */, chip::ByteSpan value)
+{
+    std::string strValue = "";
+    for (size_t i = 0; i < value.size(); i++)
+    {
+        strValue += ' ';
+        strValue += std::to_string(value.data()[i]);
+    }
+    ChipLogProgress(Zcl, "  attributeValue: (span of length %zd) %s", value.size(), strValue.c_str());
```

Comment:
That method won't print nice things for `OCTET_STRING` with non-printable characters.

Reply:
This will print the string as a series of numbers.